plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -34,7 +34,7 @@
             runtime="true"
             runtimeTypeId="com.google.cloud.tools.eclipse.appengine.standard.runtime"
             startBeforePublish="false"
-            supportsRemoteHosts="false">
+            supportsRemoteHosts="true">
```

Comment:
This is purely a UI constraint. If it's `false`, the run dialog complains with the below error message and the _Finish_ button is disabled when attempting enter non-local hosts:

![run_on_server](https://cloud.githubusercontent.com/assets/10523105/19222080/3e778e40-8e1e-11e6-96e9-55f2c0299ee5.png)

It has no function effect other than the UI constraint. The host value is simply passed on to the dev app server.